b/bazel/bazel_ubi_8_7.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : bazel
+# Version       : 6.2.0
+# Source repo   : https://github.com/bazelbuild/bazel
+# Tested on     : UBI: 8.7
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Stuti Wali <Stuti.Wali@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+PACKAGE_NAME=bazel
+PACKAGE_URL=https://github.com/bazelbuild/bazel
+
+# Default tag jgrapht
+if [ -z "$1" ]; then
+  export PACKAGE_VERSION="6.2.0"
+else
+  export PACKAGE_VERSION="$1"
+fi
+
+
+# install tools and dependent packages
+yum install -y wget git zip unzip gcc-c++ gcc make  java-11-openjdk-devel  python38
+ln -s /usr/bin/python3 /usr/bin/python
+mkdir bazel
+cd bazel
+
+#installing bazel from source
+wget https://github.com/bazelbuild/bazel/releases/download/${PACKAGE_VERSION}/bazel-${PACKAGE_VERSION}-dist.zip
+unzip bazel-${PACKAGE_VERSION}-dist.zip
+env EXTRA_BAZEL_ARGS="--tool_java_runtime_version=local_jdk" bash ./compile.sh
```

Comment:
@stutiibm  Ideally from currency perspective we want to use source tree ( branch/tag). Can you check build steps using source code instead of source zip.
cc : @seth-priya

Reply:
Okay, will check that.